fluss-server/src/main/java/org/apache/fluss/server/coordinator/CoordinatorServer.java

```diff
@@ -407,6 +399,11 @@ CompletableFuture<Void> stopServices() {
                 exception = ExceptionUtils.firstOrSuppressed(t, exception);
             }
 
+            if (ioExecutor != null) {
+                // shutdown io executor
+                ExecutorUtils.gracefulShutdown(5, TimeUnit.SECONDS, ioExecutor);
```

Comment:
We should keep the original try-catch wrapper, and handle potential exceptions?

Reply:
Miss it. I think we can follow the reasource release code pattern in here.